test/suites/snapshots.sh

```diff
@@ -256,6 +260,15 @@ snap_restore() {
       false
     fi
 
+    # Check container description is restored
+    description=$(lxc config get bar --property description)
+    if [ "${description}" != "test_description_snap0" ]; then
```

Comment:
This section is quite verbose and doesn't tell us any more than the simpler assertion:

```
[ "${description}" = "test_description_snap0" ]
```

If this evaluates to `false` then the test will fail.